frontend/package.json

```diff
@@ -28,6 +28,7 @@
     "react-error-boundary": "1.2.5",
     "react-icons": "3.9.0",
     "react-leaflet": "2.7.0",
+    "react-leaflet-control": "^2.1.2",
```

Comment:
By convention we use exact versions in `package.json` to avoid dependencies being auto-updated and breaking things. 
Please remove the `^` - thanks!